lib/nailed/default-config.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
this file and the default-colors.yml are still in the wrong place. they should go to the config folder